mmpose/structures/keypoint/transforms.py

```diff
@@ -23,10 +23,12 @@ def flip_keypoints(keypoints: np.ndarray,
             in shape (..., K, 1). Set ``None`` if the keypoint visibility is
             unavailable
         image_size (tuple): The image shape in [w, h]
-        flip_pairs (list[tuple]): The list of symmetric keypoint pairs'
-            indices
+        flip_indices (List[int]): The indices of each keypoint's symmetric
+            keypoint
         direction (str): The flip direction. Options are ``'horizontal'``,
             ``'vertical'`` and ``'diagonal'``. Defaults to ``'horizontal'``
+        inplace (bool): Whether modify ``keypoints`` and ``keypoints_visible``
+            in-place. Defaults to ``False``
```

Comment:
what does the param `inplace` use for? If only use for overwriting `results` in `RandomFlip`, I don't think it's neccessary.

Reply:
Added by mistakes, will remove it.